src/gui/Src/Gui/AppearanceDialog.ui

```diff
@@ -629,9 +629,23 @@ border-width: 1px 2px 2px 1px;</string>
       </property>
      </widget>
     </widget>
+    <widget class="QLabel" name="exampleText">
```

Comment:
I'm not sure if it's really useful to add this. I tried it but the real example is already updated live in the disassembly view and often the label can be misleading/confusing (for example with a transparent background or when there is no background color at all).

However if adding this is really necessary for you I think it should be improved by centering the text and using the same font as the disassembly, or at least a monospace font to get a better match.

Reply:
Hi, this example is used to configure some color not visible in disassembly, for example "SymbolUserTextColor"